### PR TITLE
Adapt new Android S permissions

### PR DESCRIPTION
**Pull request by clownshen**
Merge to main

### CODE REVIEW

File: NearbyConnectionsWalkieTalkie/app/build.gradle [hunk 1]
```diff
@@ -27,8 +27,8 @@ android {
 dependencies {
     implementation 'com.google.android.gms:play-services-nearby:16.0.0'
```
Comment (slavkoder) [inline NearbyConnectionsWalkieTalkie/app/build.gradle:28]: nit: please update to the latest version - 18.0.1

Comment (clownshen) [reply NearbyConnectionsWalkieTalkie/app/build.gradle]: Another commit fix this, thanks

File: NearbyConnectionsWalkieTalkie/gradle.properties [hunk 2]
```diff
@@ -9,6 +9,8 @@
 
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
+android.enableJetifier=true
+android.useAndroidX=true
 org.gradle.jvmargs=-Xmx1536m
```
Comment (Randyblo7) [inline NearbyConnectionsWalkieTalkie/gradle.properties:14]: NearbyConnectionsRockPaperScissors/app/src/main/AndroidManifest.xml

**Review by Randyblo7 — APPROVED** [review]
How

**Pull request merged**